android/src/main/java/com/internxt/mobilesdk/services/photos/PhotosApi.kt

```diff
@@ -0,0 +1,35 @@
+package com.internxt.mobilesdk.services.photos
+
+import com.internxt.mobilesdk.config.MobileSdkConfigKey
+import com.internxt.mobilesdk.config.MobileSdkConfigLoader
+import com.internxt.mobilesdk.data.photos.CreatePhotoPayload
+import com.internxt.mobilesdk.services.network.HttpClient
+import com.internxt.mobilesdk.utils.JsonUtils
+import com.internxt.mobilesdk.utils.Logger
+import okhttp3.MediaType
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+
+class PhotosApi {
+  fun createPhoto(photoPayload: CreatePhotoPayload) {
+    val createPhotoPayloadAdapter = JsonUtils.moshi.adapter(CreatePhotoPayload::class.java)
+    val base = MobileSdkConfigLoader.getConfig(MobileSdkConfigKey.PHOTOS_API_URL)
+    val url = "${base}/photo/exists"
```

Comment:
Just as a suggestion if we start changing things in the future, regarding API paths: Could be a good idea to have a function for each endpoint that returns the path? To centralize it, I mean. I say a function because of some of the dynamic params that some paths could have.

Reply:
makes sense, ok